tests/login.test.js

```diff
@@ -70,3 +75,48 @@ describe("Initialize Bynder with permanent token", () => {
     expect(bynder.api.permanentToken).toEqual("test");
   });
 });
+
+
+describe("API call headers", () => {
+  describe("with permanent token", () => {
+    beforeEach(() => {
+      bynder = new Bynder(configs);
+      axios.mockResolvedValue({});
+      bynder.api.permanentToken = "token";
+    });
+
+    afterEach(() => {
+      axios.mockRestore();
+      bynder = null;
+    });
```

Comment:
Are these having any effect? Since they should be reset in the `beforeEach` before each test.

Reply:
Nope. They're being reset on this method after each test.

Reply:
Yes, but also before each test we do:
```
      bynder = new Bynder(configs);
      axios.mockResolvedValue({});
      bynder.api.permanentToken = "token";
```

So I'd say at least the `bynder = null;` wouldn't have any effect